packit_service/models.py

```diff
@@ -366,6 +384,16 @@ def get_by_id(cls, id_: int) -> Optional["GitBranchModel"]:
         with get_sa_session() as session:
             return session.query(GitBranchModel).filter_by(id=id_).first()
 
+    def get_copr_builds(self):
+        return JobTriggerModel.get_or_create(
```

Comment:
These will create branch build and branch test run entries in the DB, even if the branch builds are not configured for a given repo. Is this okay? Shouldn't we use a plain `get` here instead?

Reply:
Okay.